external-docs/docs/guides/errors/inspect-command-errors/rdf-graph-load-failed.md

```diff
@@ -0,0 +1,9 @@
+# Error - Failed to load the RDF graph from input.
+
+## When it occurs
+
+The inspect command failed to load the input CSV-W json-ld into a RDF graph.
+
+## How to fix
+
+Make sure the input CSV-W json-ld is located at the path provided by the `TIDY_CSV_PATH` argument.
```

Comment:
TIDY_CSV_PATH change

Reply:
@robons Do you mean we need to change the `TIDY_CSV_PATH` text? Sorry, I am not understanding this comment.